<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LengthEditorWidget</class>
 <widget class="QWidget" name="LengthEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Length Editor</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>System:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="systemCombo">
     <item>
      <property name="text">
       <string>Metric</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Imperial</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Preferred unit:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="unitCombo"/>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>240</width>
       <height>25</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Length:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="qtgui::LengthDoubleSpinBox" name="lenSpin">
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>240</width>
       <height>36</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qtgui::LengthDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header location="global">qttools/gui/length_double_spinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
